internal/commands/user.go

```diff
@@ -118,12 +119,15 @@ func getUsersEmailsInConversation(
 ) (*[]string, error) {
 	var emails []string
 
+	logWriter := app.Logger.With(
+		log.NewValue("channelID", channelID),
+	)
+
 	users, err := getUsersInConversation(ctx, app, channelID)
 	if err != nil {
-		app.Logger.Error(
+		logWriter.Error(
 			ctx,
 			"command/user.getUsersEmailsInConversation",
```

Comment:
Add a log.Action("getUsersInConversation") to identify in what part of the func the error was thrown.